How can I use YCSB to evaluate Oracle's performance?

I have set the classpath included with 'ojdbc14' and I also move it to the directory ~/ycsb-0.1.4/jdbc-binding/lib.

I have create a table called 'usertable' in oracle.When use the command 
 ./ycsb load jdbc -P ../workloads/workloada -p threads=1 -p columnfamily=f1 -p recordcount=10000 -s > load.dat


issues appeared as below:

Loading workload...
Starting test.
 0 sec: 0 operations; 
Error in database operation: java.sql.SQLException: No suitable driver found for 
com.yahoo.ycsb.DBException: java.sql.SQLException: No suitable driver found for 
        at com.yahoo.ycsb.db.JdbcDBClient.init(JdbcDBClient.java:203)
        at com.yahoo.ycsb.DBWrapper.init(DBWrapper.java:63)
        at com.yahoo.ycsb.ClientThread.run(Client.java:189)
Caused by: java.sql.SQLException: No suitable driver found for 
        at java.sql.DriverManager.getConnection(DriverManager.java:602)
        at java.sql.DriverManager.getConnection(DriverManager.java:185)
        at com.yahoo.ycsb.db.JdbcDBClient.init(JdbcDBClient.java:187)
        ... 2 more
 0 sec: 0 operations; 


my conf file under /jdbc-binding:
[hadoopcdh@mastersrv conf]$ cat db.properties 
#Properties file that contains database connection information.

jdbc.driver=oracle.jdbc.driver.OracleDriver
db.url=jdbc:oracle:thin:@192.168.1.65:1521:kakou
db.user=kakou
db.passwd=kakou
[hadoopcdh@mastersrv conf]$ cat h2.properties 
#Properties file that contains database connection information.

jdbc.driver=oracle.jdbc.driver.OracleDriver
db.url=jdbc:oracle:thin:@192.168.1.65:1521:kakou
db.user=kakou
db.passwd=kakou

--- a/jdbc/src/main/java/com/yahoo/ycsb/db/JdbcDBClient.java
+++ b/jdbc/src/main/java/com/yahoo/ycsb/db/JdbcDBClient.java
@@ -245,7 +245,7 @@ public class JdbcDBClient extends DB implements JdbcDBClientConstants {
     for (int i = 0; i < insertType.numFields; i++) {
       insert.append(",?");
     }
-    insert.append(");");
+    insert.append(")");
     PreparedStatement insertStatement = getShardConnectionByKey(key).prepareStatement(insert.toString());
     PreparedStatement stmt = cachedStatements.putIfAbsent(insertType, insertStatement);
     if (stmt == null) return insertStatement;
@@ -259,7 +259,7 @@ public class JdbcDBClient extends DB implements JdbcDBClientConstants {
     read.append(" WHERE ");
     read.append(PRIMARY_KEY);
     read.append(" = ");
-    read.append("?;");
+    read.append("?");
     PreparedStatement readStatement = getShardConnectionByKey(key).prepareStatement(read.toString());
     PreparedStatement stmt = cachedStatements.putIfAbsent(readType, readStatement);
     if (stmt == null) return readStatement;
@@ -272,7 +272,7 @@ public class JdbcDBClient extends DB implements JdbcDBClientConstants {
     delete.append(deleteType.tableName);
     delete.append(" WHERE ");
     delete.append(PRIMARY_KEY);
-    delete.append(" = ?;");
+    delete.append(" = ?");
     PreparedStatement deleteStatement = getShardConnectionByKey(key).prepareStatement(delete.toString());
     PreparedStatement stmt = cachedStatements.putIfAbsent(deleteType, deleteStatement);
     if (stmt == null) return deleteStatement;
@@ -292,7 +292,7 @@ public class JdbcDBClient extends DB implements JdbcDBClientConstants {
     }
     update.append(" WHERE ");
     update.append(PRIMARY_KEY);
-    update.append(" = ?;");
+    update.append(" = ?");
     PreparedStatement insertStatement = getShardConnectionByKey(key).prepareStatement(update.toString());
     PreparedStatement stmt = cachedStatements.putIfAbsent(updateType, insertStatement);
     if (stmt == null) return insertStatement;
@@ -308,7 +308,7 @@ public class JdbcDBClient extends DB implements JdbcDBClientConstants {
     select.append(" >= ?");
     select.append(" ORDER BY ");
     select.append(PRIMARY_KEY);
-    select.append(" LIMIT ?;");
+    select.append(" LIMIT ?");
     PreparedStatement scanStatement = getShardConnectionByKey(key).prepareStatement(select.toString());
     if (this.jdbcFetchSize != null) scanStatement.setFetchSize(this.jdbcFetchSize);
     PreparedStatement stmt = cachedStatements.putIfAbsent(scanType, scanStatement);

